web/api/authentication.thrift

```diff
@@ -35,6 +35,9 @@ typedef list<SessionTokenData> SessionTokenDataList
 
 service codeCheckerAuthentication {
 
+  // Returns the CodeChecker version that is running on the server.
+  string getPackageVersion(),
```

Comment:
wasn't there already an api endpoint for this, or was the version hardcoded during the package build for the web part?

Reply:
another question is the authentication endpoint the best place for it? It is not really related to that service. Would the config service a better place for it? What do you think @csordasmarton @bruntib ?

Reply:
In the related issue I described this problem. This is not related to the config service too so I don't really know which one would be better.

Reply:
I'm just worried a bit that this will be introduced in another service even if it is already there in the other two, couldn't we use it from the configuration? I know it does not fit everywhere but at least it is already there and no new endpoint introduction is needed.

Reply:
Something is definitely an oversight here, this might have been my mistake back when authentication was implemented over the whole product stuff.

Reply:
Maybe best if we create a dedicated "Version" endpoint, move it there, and deprecate the API calls to the old endpoints? That would create the least amount of confusion...

Reply:
New API endpoint (Version) is introduced with a getPackageVersion function.